.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Smoke Tests
```

Comment:
Why isn't this called "go get -u test" or something similar?

Reply:
I'm expecting to put more smoke tests cases as jobs of this workflow.
The next one I have in mind is `go mod vendor` where we used to have many issues in the past (mostly before go modules existed ^^') where some files were not vendored and it was leading to compilation issues too.